internal/locate/region_request.go

```diff
@@ -829,7 +829,7 @@ func (s *RegionRequestSender) SendReqCtx(
 				if err := s.replicaSelector.backoffOnNoCandidate(bo); err != nil {
 					return nil, nil, retryTimes, err
 				}
-				if cost := time.Since(startTime); cost > slowLogSendReqTime || cost > timeout {
+				if cost := time.Since(startTime); cost > slowLogSendReqTime || cost > timeout || bo.GetTotalSleep() > 1000 {
```

Comment:
Is it necessary to make the backoff time `1000` configurable?

Reply:
Add a config or session variable need PM agree, I don't think this is a strong demand currently.

Reply:
I thinks it's ok to have a configuration for the log-threshold, but it introduces a lot of extra work like discussing with PM..